deployability/modules/testing/models.py

```diff
@@ -27,16 +25,17 @@ def validate_tests(cls, value) -> list[str]:
             value = value.split(',')
         return value
 
-    @field_validator('inventory', mode='before')
```

Comment:
This decorator is required, the validator won't work without it

Reply:
Done https://github.com/wazuh/wazuh-qa/pull/4884/commits/507e376cb389ea15245b6538a710341f4cdf0ec1